src/Sulu/Bundle/WebsiteBundle/Resources/views/Extension/seo.html.twig

```diff
@@ -28,8 +28,8 @@
 
 {% set seoCanonical = seo.canonicalUrl|default() %}
 
-{%- if not seoCanonical and shadowBaseLocale and urls[shadowBaseLocale]|default() %}
-    {% set seoCanonical = sulu_content_path(urls[shadowBaseLocale], null, shadowBaseLocale) %}
+{%- if not seoCanonical and shadowBaseLocale and localizations[shadowBaseLocale]|default() %}
```

Comment:
think here is missing or urls missing:

```
%- if not seoCanonical and shadowBaseLocale and (localizations[shadowBaseLocale]|default() or urls[shadowBaseLocale]|default()) %}
```

Reply:
I think I would do something like this on the top of this file so we can easily remove it in future and throw also a deprecated message in this twig file:

```twig
{% if localizations is not defined %}
    {% deprecated 'The "urls" variable is deprecated, set "localizations" instead in the "seo.html.twig".' %}
    {% set localizations = {} %}
    {% for locale, url in urls %}
        {% set localizations = localizations|merge({
            (locale): {
                locale: locale, 
                url: sulu_content_path(url, null, locale),
            }
        }) %}
    {% endfor %}
{% endif %}
```